docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -0,0 +1,328 @@
+# Underscored Attributes Reference
+
+**WARNING:** This information is provided primarily for compiler and standard
+library developers. Usage of these attributes outside of the Swift monorepo
+is STRONGLY DISCOURAGED.
+
+The Swift reference has a chapter discussing [stable attributes][Attributes].
+This document is intended to serve as a counter-part describing underscored
+attributes, whose semantics are subject to change and most likely need to
+go through the Swift evolution process before being stabilized.
+
+[Attributes]: https://docs.swift.org/swift-book/ReferenceManual/Attributes.html
+
+The attributes are organized in alphabetical order.
+
+## `@_alignment`
```

Comment:
An attribute that allows the user to control the alignment of a type. The alignment value specified must be a power of two, and cannot be less than the "natural" alignment of the type that would otherwise be used by the Swift ABI. This attribute is intended for the SIMD types in the standard library which use it to increase their internal storage alignment to at least 16 bytes.